definitions/marts/looker_studio_marts/ecf/ls_ecf_2021_declarations_in_milestone_period.sqlx

```diff
@@ -0,0 +1,189 @@
+config {
+    type: "table",
+    assertions: {
+        uniqueKey: ["declaration_id"]
+    },
+    bigquery: {
+        partitionBy: "DATE(declaration_date)",
+        clusterBy: ["role"]
+    },
+    description: "This mart holds all funded (paid, payable, eligible) ECF declarations for participants in the 2021 cohort with multiple in any one Milestone Period. \n \
+    Objective: \n \
+      To identify all declarations that fall within the same milestone period for review. \n \
+    Dependencies: \n \
+      static_tables.ecf_2021_milestone_periods \n \
+      dataform.ls_declarations_provider_names",
+    columns: {
+        user_id: "This comes from the teacher profile associated with the participant profile. User ID also exists independently in other tables.",
+        participant_profile_id: "The ID from a participant's profile in the CPD service. Participant profiles are only generated when an application is accepted by a lead provider. There is one profile per accepted application. An NPQ participant will have a different participant_profile_id for each accepted NPQ. Their participant_profile_id should match the application_id and should match the participant_profile_id available in the declarations_latest table. While their participant_profile_id changes across NPQs, their participant_identity_id will remain the same across their participant profiles so long as they have signed up with the same email.",

```

Comment:
This isn't targeted to this table, there's irrelevant content, we don't need to know about NPQs or or connecting to declarations because that's not within the remit of this mart.